OpenUtau/Strings/Strings.it-IT.axaml

```diff
@@ -223,10 +223,10 @@ Attenzione: questa opzione rimuove tutti i preset salvati.</system:String>
   <system:String x:Key="pianoroll.toggle.vibrato">Mostra Vibrato (U)</system:String>
   <system:String x:Key="pianoroll.toggle.waveform">Mostra Waveform (W)</system:String>
   <system:String x:Key="pianoroll.tool.drawpitch">Strumento Disegna Pitch (4)</system:String>
```

Comment:
This line has an unnecessary closing tag, remove ``</system:String>`` and it should work.